README.md

```diff
@@ -68,8 +70,14 @@ Please note that these plugins library projects are not imported as project modu
 
 ## Using in Android Studio
 Disable instant run feature in Android Studio
-# Debugging native code
-Add 
+
+# Debugging or modifying native code
+To modify or debug native code it is necessary to delete folder `AndroidSeed/tealeaf/src/main/libs/` with its content and to build game in debug mode
```

Comment:
This path is wrong I think @CodeCombiner 
It should be AndroidSeed/tealeaf/src/main/jni/libs

Also. Only libtealeaf.so will be rebuilt or everylibrary will be rebuilt ?

Reply:
Please don't remove `AndroidSeed/tealeaf/src/main/jni/libs` , this path is a part of native source code. You need to remove `AndroidSeed/tealeaf/src/main/libs/`

Reply:
I will add 1 line to notify that `AndroidSeed/tealeaf/src/main/obj/` shell also be removed

Reply:
Only libtealeaf.so shared library will be rebuilt. This shared library includes several static libraries which are also rebuilt during libtealeaf.so building

Reply:
I don't have  a libs folder under `AndroidSeed/tealeaf/src/main/`

Reply:
I think they were in gitignore when I committed them or I removed them during last commit. I will commit again the release version of shared libraries and objects. I think having release library by default is the best